### DISCUSSION TITLE
模型导出耗时多长？

### DISCUSSION THREAD

**Discussion by gubinjie** [PaddlePaddle/PaddleOCR#12865] [Q&A]
你好，我导出模型跑了几个小时是否正常？
python tools/export_model.py -c ./configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_student.yml --output_dir=./inference_model -o weights=./output/dianbiao_ch_PP-OCR_V3_det/latest.pdparams --export_serving_model=True
这是这个命令，感觉几个小时是否不太符合期待？但是后台一直再跑，也没有错误信息

**Comment by SWHL** [comment]
如果问题仍然存在，请尝试用这个：[link](https://rapidai.github.io/RapidOCRDocs/blog/2022/09/27/%E8%BD%AC%E6%8D%A2paddleocr%E6%A8%A1%E5%9E%8B%E4%B8%BAonnx/)
